### DISCUSSION TITLE
System HTTP task does not retry

### DISCUSSION THREAD

**Discussion by dcore94** [Netflix/conductor#2538] [Q&A]
Even if I have the following definition:
{
            "name" : "http",
            "taskReferenceName" : "get_workspace",
            "type" : "HTTP",
            "inputParameters" : {
              "http_request": {
                 "uri" : "${init.input.storagehub}/workspace",
                 "method" : "GET"
              }
            },
            "retryCount" : 5,
            "retryDelaySeconds": 10
}
The task still fails on occasional ReadTimeoutExceptions blocking the whole workflow in FAILED status. I would expect it to retry 5 times within the following minute....

I don't see why since the code at [1] looks ok to me... any explanation? Thanks!

[1] https://github.com/Netflix/conductor/blob/38379b84d05f8134544dd8a3a0c91f7fd6ad73ef/contribs/src/main/java/com/netflix/conductor/contribs/tasks/http/HttpTask.java#L129

**Comment by jxu-nflx** [comment]
Hello @dcore94 Can you provide more details of the failed task, and any logs that indicates the task is not retried?
